Client/src/components/contanier/Auth/index.js

```diff
@@ -1,27 +1,136 @@
-import React, { Component } from 'react'
+import React, {
+  Component
+} from 'react'
+import axios from 'axios';
 
 import Button from '../../common/Button'
 import LabeledInput from '../../common/LabeledInput'
 import logo from '../../assets/Image/amideasblue.png'
-import { firstLable, secondLable } from './staticdata'
 import './index.css'
 
 export default class Login extends Component {
 
+  state = {
+    id: '',
+    password: '',
+    loginError: '',
+    role: '',
+  };
+
+  handleChange = ({
+    target: {
+      name,
+      value
+    }
+  }) => {
+    this.setState({
+      [name]: value.trim()
+    });
+  };
+
   handleClick = (e) => {
-    e.preventdefault()
+    e.preventDefault()
+    const {
+      id,
+      password
+    } = this.state;
+    if (!id || !password) {
+      this.setState({
+        loginError: 'All Fields Requier'
+      })
+    } else if (id.length === 0) {
+      this.setState({
+        loginError: 'ID At least Five Numbers'
+      })
+    } else if (password.length < 5) {
+      this.setState({
+        loginError: 'Password At Least Five'
+      })
+    } else {
+      this.setState({
+        loginError: ''
+      });
+      axios
+        .post('/api/v1/login', {
+          id,
+          password
+        })
+        .then(res => {
+          if (res.data.error) {
+            this.setState({
+              loginError: res.error
+            })
+          } else if (res.data.data.length) {
+            if (res.data.data[0].id.toString().length === 5) {
+              this.props.handleLogin(res.data.data[0].id)
+              this.props.history.push('/student/courses')
+            } else {
+              this.props.handleLogin(res.data.data[0].id)
+              this.props.history.push('/staff/courses')
+            }
+          } else {
+            this.setState({
+              loginError: 'Check your ID or Password'
+            })
+          }
+        })
+        .catch((e) => {
+          this.setState({
+            loginError: 'Try Againe'
+          })
+        })
+    }
   }
-  render() {
 
-    return (
-      <div className='login'>
-        <img src={logo} alt="amideast logo" className='login-logo' /> 
-        <div className='mobile-container'>
-          <LabeledInput {...firstLable}/>
-          <LabeledInput {...secondLable}/>  
-        </div> 
-        <Button content='Login' className='login-btn' onClick={this.handleClick}/>    
-      </div>
+  render() {
+    return ( <
+      div className = 'login' >
+      <
+      img src = {
+        logo
+      }
+      alt = "amideast logo"
+      className = 'login-logo' / >
+      <
+      div className = 'mobile-container' >
+      <
+      LabeledInput LabeledInputClassName = 'container'
```

Comment:
@AbdallahAmmar96 If you have time, I think this code would be easier to read if you put all of the opening / closing brackets onto the same line as the tag and indented child tags, so for example change it from this:
```
<
div className='login>
<
img src=...
```
to this
```
<div className='login>
  <img src=...
```